spec-main/api-net-spec.ts

```diff
@@ -592,8 +592,9 @@ describe('net module', () => {
           url: `${serverUrl}`,
           name: 'test',
           value: '11111',
-          expirationDate: 0
-        }).then(() => { // resolved
+          expirationDate: new Date().getTime() + 30000
+        }).then(() => customSession.cookies.flushStore()
```

Comment:
This is kind of awkward syntax, if you feel like it it would be great to convert this test to use `async`! But not needed for the PR.